R/pkg/R/generics.R

```diff
@@ -20,157 +20,196 @@
 # @rdname aggregateRDD
 # @seealso reduce
 # @export
+# @note since 1.5.0
```

Comment:
I'm not sure we need to put @note on the generic. Basically this merges with the other roxygen block because of the @rdname tag, so it should already has one that. More concerning is whether doing it here would double it in the output html file - could you check?